src/main/java/seedu/address/logic/commands/ExportCommand.java

```diff
@@ -0,0 +1,148 @@
+
+    public final String fileTypeCsv = "CSV";
+
+    private final String csvHeaders = "Name,Phone No,Email,Address,Tags,Notes\n";
+
+    private final Path exportCsvPath = Paths.get("exports" , "bizbook.csv");
+
+    private final String fileType;
+
+    /**
+     * Creates an ExportCsvCommand.
+     *
+     * @param fileType to export the data into.
+     */
+    public ExportCommand(String fileType) {
+        requireNonNull(fileType);
+        this.fileType = fileType;
+    }
+
+    @Override
+    public CommandResult execute(Model model) throws CommandException {
+        requireNonNull(model);
+
+        try {
+            if (fileType.equals(fileTypeCsv)) {
+                exportToCsv(model);
+            }
+        } catch (IOException io) {
+            throw new CommandException(MESSAGE_FAILURE);
+        }
+
+        return new CommandResult(String.format(MESSAGE_SUCCESS, fileType));
+    }
+
+    /**
+     * Exports the contact list into a csv file.
+     * Iterate through every person, convert them to a csv format and
+     * write into the csv file.
+     *
+     * @throws IOException if an error occurs when creating the csv file.
+     */
+    private void exportToCsv(Model model) throws IOException {
+        if (!Files.exists(exportCsvPath.getParent())) {
+            Files.createDirectories(exportCsvPath.getName(0));
+        }
+        // Create a csv file to save the tasks
+        File dataFile = new File(exportCsvPath.toString());
+        FileWriter fw = new FileWriter(dataFile);
+
+        fw.write(this.csvHeaders);
+
+        ObservableList<Person> personList = model.getFilteredPersonList();
+        for (Person person : personList) {
+            String personData = this.toCsvString(person);
+            fw.write(personData + "\n");
+        }
+        fw.close();
+    }
+
+    /**
+     * Converts a {@code Person} object into a csv format.
+     *
+     * @param person to be encoded into a csv format.
+     * @return A csv representation of the {@Code Person} object.
+     */
+    public String toCsvString(Person person) {
+        StringJoiner sj = new StringJoiner(",");
+
+        sj.add(person.getName().fullName);
+        sj.add(person.getPhone().value);
+        sj.add(person.getEmail().value);
+
+        // Prevent excel from sepreating entries due to commas
+        String address = "\"" + person.getAddress().value + "\"";
+        sj.add(address);
+
+        List<String> tags = new ArrayList<>();
+        List<String> notes = new ArrayList<>();
+
+        tags.addAll(person.getTags().stream()
+            .map((tag) -> tag.toString())
+            .toList());
+
+        notes.addAll(person.getNotes().stream()
+            .map((note) -> note.toString())
+            .toList());
+
+        sj.add("\"" + tags.toString().replaceAll("[\\[\\]]", "") + "\"");
+        sj.add("\"" + notes.toString().replaceAll("[\\[\\]]", "") + "\"");
```

Comment:
This doesn't seem needed since tags and notes are alphanumeric

Reply:
For this er the `toString` method will have a  `[ ]` encased for each tag and note so for readability for the user I removed it.

As for adding the " ", it is to accept the commas that separate each tag or note if not then it will have uneven columns

Reply:
I just realised why this is needed: because you are converting a `List` to string and because note adds the square braces when you use `toString`. Maybe you can combine them instead?

```java
String notes = person.getNotes().stream()
    .map(Note::getNote).collect(Collectors.joining(","));
```